server/build/webpack.js

```diff
@@ -39,6 +44,10 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       log: false,
       // required not to cache removed files
       useHashIndex: false
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Am I correct the `minChunks` setting causes to load some modules multiple times ?